examples/stable_diffusion/convert_sd_adapter_to_peft.py

```diff
@@ -279,12 +285,21 @@ def detect_adapter_type(keys: List[str]) -> PeftType:
             elif kohya_type == "hada_w2_b":
                 adapter_info[model_type][peft_key].hada_w2_b = tensor
                 adapter_info[model_type][peft_key].rank = tensor.shape[0]
-            elif kohya_type == "hada_t1":
-                adapter_info[model_type][peft_key].hada_t1 = tensor
-                adapter_info[model_type][peft_key].rank = tensor.shape[0]
-            elif kohya_type == "hada_t2":
-                adapter_info[model_type][peft_key].hada_t2 = tensor
-                adapter_info[model_type][peft_key].rank = tensor.shape[0]
+            elif kohya_type in {"hada_t1", "hada_t2"}:
+                if args.loha_weights_fix:
+                    if kohya_type == "hada_t1":
+                        # Right now there exists a bug, so we need to rewrite hada_t2 tensor with hada_t1 info
```

Comment:
Let's not use word as "Right now", because they will be outdated quickly. Maybe something along the lines of: "This code block accommodates a bug that existed for some time that resulted in accidentally using hada_t1 weight for hada_t2, see ...".

WDYT?